examples/platforms/efr32/radio.c

```diff
@@ -361,13 +369,13 @@ otRadioFrame *otPlatRadioGetTransmitBuffer(otInstance *aInstance)
 int8_t otPlatRadioGetRssi(otInstance *aInstance)
 {
     (void)aInstance;
-    return (uint8_t)(RAIL_RxGetRSSI() >> 2);
+    return (int8_t)(RAIL_GetAverageRssi(sRailHandle) >> 2);
 }
 
 otRadioCaps otPlatRadioGetCaps(otInstance *aInstance)
 {
     (void)aInstance;
-    return OT_RADIO_CAPS_ACK_TIMEOUT;
+    return OT_RADIO_CAPS_ACK_TIMEOUT | OT_RADIO_CAPS_CSMA_BACKOFF;
```

Comment:
Shall OT_RADIO_CAPS_TRANSMIT_RETRIES be added as well?

Reply:
Does RAIL implement automatic retransmissions on no ack?

Reply:
I didn't find any info about retransmissions. So, most probably doesn't implement.